drafts/one-percent-for-open-source.rst

```diff
@@ -0,0 +1,111 @@
+.. post::
+   :tags: python, opensource, sustainability, sustain
+
+One Percent for Open Source
+===========================
+
+The open source ecosystem is the `most valuable part`_ of the software industry today.
+All of the companies and technologies that we build today are enabled by open source software.
+From the programming languages to the web frameworks,
+the operating system to the cryptography,
+all of it is open source.
+
+Open source has a `funding problem`_.
+There are many `warning signs`_ that open source is an ecosystem in danger.
+
+Open source is maintained for "free" by people all over the world,
+enabled by the internet (which is also built on open source).
+Many people doing this work are burning out,
+giving up,
+or just never standing in the first place.
```

Comment:
There's a few different categories here - there's freeloaders, there's people who don't think they have the capability, and there's people who literally can't due to other obligations (e.g., having family/carer obligations outside work hours). "Never standing in the first place" is a little pejorative; but the pejorative sense only applies to the first category.

Reply:
I meant to say "starting", which makes a bit more sense!

Reply:
I'll change to "getting involved", which feels better.